doc/administrator_guide.md

```diff
@@ -5,32 +5,24 @@ draft: false
 wieght: 40
 description: "Deploying, maintaining, and upgrading a KCIDB service installation"
 ---
-Kcidb infrastructure is mostly based on Google Cloud services at the moment:
-
-    === Hosts ===  ======================= Google Cloud Project ========================
-
-    ~ Webserver ~                                                    ~~~~ BigQuery ~~~~~
-    kcidb-grafana <-------------------------------------------------  .. kcidb_XX ..
-                                                                     :   checkouts  :
-    ~~ Client ~~~                                                    :   builds     :
-    kcidb-query <--------------------------------------------------- :   tests      :
-                                                                      ''''''''''''''
-                    ~~ Pub/Sub ~~       ~~~~ Cloud Functions ~~~~            ^
-                    kcidb_trigger ----------.                                |
-                                            v                                |
-    kcidb-submit -> kcidb_new --------> kcidb_load_queue --------------------'
-                                            |
-                          .-----------------'
-                          v                                          ~~~~ Firestore ~~~~
-                    kcidb_updated ----> kcidb_spool_notifications -> notifications
-                                                                           |
-                                                   .-----------------------'
-                                                   |
-                                                   v                 ~ Secret Manager ~~
-                                        kcidb_send_notification <--- kcidb_smtp_password
-                                                   |
-                                                   |                 ~~~~~~ GMail ~~~~~~
-                                                   `---------------> bot@kernelci.org
+```mermaid
+graph TD
+  Webserver["Webserver"] --> BigQuery["BigQuery"]
+  Client["~~ Client ~~"] --> BigQuery
+  kcidb-query -->|Queries| BigQuery
+  BigQuery -->|Reports| kcidb-grafana["kcidb-grafana"]
+  
+  kcidb-submit --> kcidb_new["kcidb_new Pub/Sub topic"]
```

Comment:
Is there a way to have different boxes for different things? Like have Pub/Sub topics one color, Cloud Functions another, and so on? Having to spell out the object type each time would be cumbersome and make reading the diagram harder. What expressive options do we have with Mermaid in general?

Reply:
Yes there are ways make containers but I am not sure about the color coding, have to look.